packages/react-components/react-toolbar/src/components/Toolbar/useToolbarContextValues.tsx

```diff
@@ -1,10 +1,11 @@
 import type { ToolbarContextValue, ToolbarContextValues, ToolbarState } from './Toolbar.types';
 
 export function useToolbarContextValues_unstable(state: ToolbarState): ToolbarContextValues {
-  const { size, handleToggleButton } = state;
+  const { size, handleToggleButton, vertical } = state;
   // This context is created with "@fluentui/react-context-selector", these is no sense to memoize it
   const toolbar: ToolbarContextValue = {
     size,
+    vertical,
     handleToggleButton,
   };
```

Comment:
You're actually not using `@fluentui/react-context-selector` at `ToolbarContext` creation,  so perhaps it would be better to start memoizing the value.

Reply:
Good catch, the context value should be memoized

Reply:
I think this address what you ask: https://github.com/microsoft/fluentui/pull/24940/commits/23f846ced517e870811d7fb7da90f05ef95074d6 If not, please let me know.